### DISCUSSION TITLE
rAudio 20240107: Drop-out during the first seconds of playlist with heavy CPU-load

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1554] [Fixed Issues]
As the title says, during the first 2, 3 seconds there are drop-outs. Responsible for the drop-outs are a short 100% load of 3 of the 4 CPUs of my quad-core RPi 3B. 
What's the reason of the processor load?

**Comment by ErikDorstel** [comment]
I can confirm the dropouts. They appear at the beginning of MP3 files. FLAC files are not affected.

Hardware: RPi 4B with I2S Allo Boss DAC and NVMe M.2 SSD over USB

I couldn't observe a CPU load of 100%. Which tool was used to determine the load?

**Reply by klassikmann** [reply]
@ErikDorstel 
There are FLAC files in my database only, i.e. no MP3.
Observation: Only the first title of a playlist seems to be affected by the drop-outs (observation not yet fully confirmed).
To determine the CPU load I'm using "top" or "htop" (to be installed separately) with a short delay between updates ("top -d 0.3" or "htop -d 3").

**Comment by klassikmann** [comment]
rAudio 20240108: A new version has been released superseding 20240107. The release notes do not mention the (nasty) drop-out issue.
The CPU load is still very high at starting a playlist but not quite as high as yesterday. Some albums start correctly today, others do not.
My observation ("Observation: Only the first title of a playlist seems to be affected by the drop-outs") could be confirmed, the CPU load at a track boundary is medium.

**Comment by rern** [comment]
Guys,

System wide upgrade might have something to do with the drop-outs/glitches. Please try the [latest image](https://github.com/rern/rAudio/releases/tag/i20240108).

**Reply by klassikmann** [reply]
Latest image: Same problems (drop-outs) as before. 
The new image is also based on the new kernel 6.6. Perhaps the upgrade from 6.1 to 6.6 is the culprit.

**Comment by klassikmann** [comment]
I found the culprit: Uncheck Features > For Browsers
Later more details.

**Comment by klassikmann** [comment]
> I found the culprit: Uncheck Features > For Browsers Later more details.

What I did:
1.  I made a system based on rAudio-64bit-20231128 and I restored yesterday's backup.
Result: Drop-outs => Problem is **not** related to kernel 6.6.

2.  I made a system based on the same older rAudio-64bit-20231128 and I restored an older backup.
Result: No Drop-outs

3. Thinking about the differences regarding settings and features in the two backups.
Result: I activated "Features > For Browsers" at New Year's Eve because I was not at home but I wanted to use my rAudio installation remote via VPN and port 8000.
Setting "Features > For Browsers" is only meaningful (in my environment) if afar from my living room.

**Reply by ErikDorstel** [reply]
That's strange. I've never used "MTD HTTP" and had the drop outs with release 20240107. (or was it 20240105? I'm unsure... unfortunately the list of releases is incomplete)

For me the drop outs disappeared with release 20240108 and also with i20240108.

**Reply by cmh714** [reply]
Well I havent experienced the issue, but I have been playing with Tailscale to "vpn" back into my rAudio system.....works great.
